### PR TITLE
Do not require SELECTED_NETWORKS env when importing networks package

### PR DESCRIPTION
**Pull request by lukaszcl**
Related PRs that will be merged after this one:
- https://github.com/smartcontractkit/chainlink/pull/11103
- https://github.com/smartcontractkit/mercury-pipeline/pull/431

### CODE REVIEW

**Review by gheorghestrimtu — APPROVED** [review]

**Review by gheorghestrimtu — APPROVED** [review]

File: networks/known_networks.go [hunk 1]
```diff
@@ -625,113 +618,82 @@ var (
...
+	}
+	networks := make([]blockchain.EVMNetwork, 0)
+	for i := range networkKeys {
+		var walletKeys, httpUrls, wsUrls []string
+		if !strings.Contains(networkKeys[i], "SIMULATED") {
+			// Get network RPC WS URL from env var
+			wsEnvVar := fmt.Sprintf("%s_URLS", networkKeys[i])
+			wsEnvVal := os.Getenv(wsEnvVar)
+			if wsEnvVal == "" {
+				panic(errors.Errorf("set %s env var", wsEnvVar))
```
[the remark this answers is not in the document]
Comment (lukaszcl) [reply networks/known_networks.go:640]: @Tofel I see the default urls are only used for "simulated" networks. And, for "simulated" this code will not be executed `if !strings.Contains(networkKeys[i], "SIMULATED")`

[the remark this answers is not in the document]
Comment (lukaszcl) [reply networks/known_networks.go:640]: @Tofel but, I see I need to update the code anyway as the simulated networks will be overwritten with []string

[the remark this answers is not in the document]
Comment (lukaszcl) [reply networks/known_networks.go:640]: @Tofel I pushed a new [commit](https://github.com/smartcontractkit/chainlink-testing-framework/pull/748/commits/0c3a0475692e93ca7590b8cd229a5d096c5212d6)

**Review by Tofel — COMMENTED** [review]
👍

**Pull request merged**